README.md

```diff
@@ -78,56 +80,93 @@ DATABASES = {
+  Django will override values in `settings` with those from `settings_local.py`, if `settings_local.py` exists.
 
+3. Run the database migration scripts
+  ```
+  python manage.py migrate
+  ```
+  
+  OSX: If you get the following error `django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module: dlopen(_mysql.so, 2): Library not loaded: libssl.1.0.0.dylib`
+  
+  Then, you need to add openssl to your `DYLD_LIBRARY_PATH`:
+  1. Go to `/usr/local/Cellar/openssl/`, and locate your directory (e.g. 1.0.2d_1)
+  2. Add the following to your `~/.bash_profile`:
+   ```
+   export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:/usr/local/Cellar/openssl/1.0.2d_1/lib
+   ```
 
-## Run the tests
 
-    $ make test
+### Validate your install
 
-It should load and clean some files in a few seconds.
+#### Run the tests
 
-## Download the data
+  ```
+  python manage.py test
+  ```
 
-First, a basic data check to make sure things are working:
+  It should load and clean some files in a few seconds.
+  
+  Note: if this fails with an SSL error and you are using conda/miniconda, use virtualenv instead. See [this link](https://groups.google.com/a/continuum.io/forum/#!topic/conda/Fqv93VKQXAc) for details about the conda issue. 
 
-    $ python manage.py downloadcalaccessrawdata --use-test-data
+#### Run with test data
 
-### Zipcode/metro data
+  A basic data check to make sure things are working:
 
-    $ python manage.py downloadzipcodedata
+  ```
+  python manage.py downloadcalaccessrawdata --use-test-data
+  ```
 
-### Netfile
+
+### Download data
+
+#### Zipcode/metro data
+
+  ```
+  python manage.py downloadzipcodedata
+  ```
+
+#### Netfile
 
 Netfile contains campaign finance data for a number of jurisdictions. Not all
 jurisdictions will have data.
 
-    $ python manage.py downloadnetfilerawdata
+  ```
+  python manage.py downloadnetfilerawdata
+  ```
 
-### Cal-Access
+#### Cal-Access
 
 Cal-Access is the state data. It's ~750MB of data and takes over an hour to
 trim, clean and process.
 
-    $ python manage.py downloadcalaccessrawdata
+  ```
+  python manage.py downloadcalaccessrawdata
+  ```
+
+
+### Run the server
 
-## Deploying
+  To run for the purposes of development, accessing Django's admin interface:
 
-```
-ssh opencal.opendisclosure.io /usr/local/bin/deploy-backend
-```
+  ```
+  python manage.py createsuperuser  # create a username/password for yourself
+  python manage.py runserver
+  ```
 
-To run for the purposes of development, accessing Django's admin interface:
+  Then go to http://127.0.0.1:8000/admin to log in and see data.
 
-```
-python manage.py createsuperuser
-< and create a username/password for yourself>
 
-python manage.py runserver
-```
+## Deploying the app
 
-Then go to http://127.0.0.1:8000/admin to log in and see data.
+  For deployment to the official website:
 
+  ```
+  ssh opencal.opendisclosure.io /usr/local/bin/deploy-backend
+  ```
```

Comment:
Another nit: The two-space indentation of each section's contents seems unnecessary.